src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/PlayerRequestResolver.java

```diff
@@ -72,7 +72,7 @@ public boolean canResolve(@NotNull final IRequestManager manager, final IRequest
     }
 
     @Override
-    public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequest request) throws RuntimeException
+    public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequest<?> request) throws RuntimeException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.RuntimeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)